src/Pact/LSP/PactTrace.hs

```diff
@@ -43,12 +43,11 @@ parseEntry = do
       , string' "OutputFailure" $> DsError
       , string' "Trace"         $> DsInfo
       , string' "Error"         $> DsError
-      , string' " error"        $> DsError
       ]
     toRange ln col = let ln' = ln-1
                      in Range (Position ln' col) (Position ln' (col+3))
     pInfo = do
-      fp <- many1 (noneOf ":\n")
+      fp <- many1 (noneOf " :\n")
```

Comment:
we can make this a little more robust by doing something like 

```haskell
fp <- many1 $ spaces >> noneOf ":\n"
```

Just to make it explicit that we're  skipping whitespace here.

Reply:
The intention here is to fail parsing, as the filepath should begin the line. Multiline errors are intended by spaces.